Event Signup/EventSignup.html

```diff
@@ -4,8 +4,46 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Event Signup</title>
+    <!--<link rel = "stylesheet" href="styles.css">-->
 </head>
 <body>
-    
+    <!--Header Form template.-->
+    <header>
+        <!--Header Content Here.-->
+    </header>
+    <!--Navigation Form Template-->
+    <nav>
+        <!--Navigation Content Here.-->
```

Comment:
Use template.html to fill in the header, nav and footer for your own html file

Reply:
Beautiful